### PR TITLE
fix: GEOT-318: restructure html css group page fix notifications sidebar

### PR DESCRIPTION
**Pull request by ra8ga**

### CODE REVIEW

File: packages/ui/PostSingle/index.tsx [hunk 1]
```diff
@@ -74,14 +74,30 @@ export const PostSingle: FC<{
       </div>
       <div className={styles.desc}>
         <ReactMarkdown className={styles.text} source={text} />
+        <ul className={styles.images}>
```
Comment (mariusz-kabala) [inline packages/ui/PostSingle/index.tsx:77]: this mock does not make sense, images should use <image /> not bg, and should be wrapped with <a /> to full size image

File: packages/app-andrew/src/pages/_app.tsx [hunk 2]
```diff
@@ -10,7 +10,6 @@ import { NotificationsActive } from 'components/commons/NotificationsActive'
 import { init, initAuthSession } from '@gtms/state-user'
 import { init as initWPN } from '@gtms/state-notification'
 import { LoginWindow } from 'components/commons/LoginWindow'
-import { NotificationsSidebar } from 'components/commons/NotificationsSidebar'
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/_app.tsx:13]: why? notifications must be global, keep it in app

File: packages/app-andrew/src/pages/group/[slug].tsx [hunk 3]
```diff
@@ -95,26 +96,27 @@ const GroupPage: NextPage<GroupPageProps> = (props) => {
   }, [])
 
   return (
-    <div className={styles.wrapper}>
-      {state.isLoading && <Spinner />}
+    <>
+      <NotificationsSidebar />
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/group/[slug].tsx:100]: remove notifications from group

File: packages/ui/PostSingle/index.tsx (hunk 1 above)
Comment (ra8ga) [reply packages/ui/PostSingle/index.tsx]: fixed

File: packages/app-andrew/src/pages/_app.tsx (hunk 2 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/_app.tsx:13]: removed

File: packages/app-andrew/src/pages/group/[slug].tsx (hunk 3 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/group/[slug].tsx]: removed

**Pull request merged**